codice/ddf
# DDF-2459: Content directory monitors now keep working after system restart

#### What does this PR do?
CDMs now wait for the "content" CamelComponent before adding routes.
* Have the CDM retry adding routes for a given period of time.
* Wait for the content CamelComponent instead of recursively retrying on exceptions.

#### Who is reviewing it?
@shaundmorris 
@jlcsmith

#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).
@lessarderic 
@pklinef

#### How should this be tested?
* Ensure the PR passes a full build (hero)
* Setup a monitored directory and ensure content is still ingested after a system reboot

#### Checklist:
